test_scripts/Resumption/InteriorVehicleData/002_InteriorVD_resumption_one_moduleType_with_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after transport disconnect
+--  in case app was subscribed to one module with moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 via GetInteriorVehicleData(module_1, moduleId)
+--
+-- Sequence:
+-- 1. Transport disconnect and reconnect are performed
+-- 2. App starts registration with actual hashId after unexpected disconnect
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, module_1, moduleId) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
please change all RAI(SUCCESS) to RAI(success=true, result code = SUCCESS)

Reply:
Done